status.tmpl

```diff
@@ -0,0 +1,97 @@
+<html>
+  <head>
+    <style type="text/css">
```

Comment:
It's less than 100 lines =) If we do that, we have to chunk the CSS file into the binary too, and then write an HTTP handler for it...